oshi-core/src/main/java/oshi/util/platform/windows/WmiQueryHandler.java

```diff
@@ -39,12 +39,14 @@
 import com.sun.jna.platform.win32.COM.Wbemcli;
 import com.sun.jna.platform.win32.COM.WbemcliUtil;
 
+import oshi.util.ConfigUtil;
+
 public class WmiQueryHandler {
 
     private static final Logger LOG = LoggerFactory.getLogger(WmiQueryHandler.class);
 
     // Timeout for WMI queries
-    private int wmiTimeout = Wbemcli.WBEM_INFINITE;
+    private int wmiTimeout = ConfigUtil.getInteger("oshi.util.wmi.timeout");
```

Comment:
I'd like to set the default in this class, either through getting the string and parsing here (with default) or passing the default to the getter.   Do we want to sanity check (positive value)?

Reply:
Should `getWmiTimeout()` and `setWmiTimeout()` be removed now?

Reply:
> Should `getWmiTimeout()` and `setWmiTimeout()` be removed now?

No!  The setter is necessary to override the default on an individual query basis.